lab/src/Microsoft.ServiceModel.Syndication/src/SyndicationFeed.cs

```diff
@@ -31,13 +28,101 @@ public class SyndicationFeed : IExtensibleSyndicationObject
         private ExtensibleSyndicationObject _extensions = new ExtensibleSyndicationObject();
         private string _generator;
         private string _id;
+
         private Uri _imageUrl;
+        private TextSyndicationContent _imageTitle;
+        private Uri _imageLink;
+        
         private IEnumerable<SyndicationItem> _items;
         private string _language;
         private DateTimeOffset _lastUpdatedTime;
         private Collection<SyndicationLink> _links;
         private TextSyndicationContent _title;
 
+        //new fields for new optional RSS tags
+        private SyndicationLink _documentation;
```

Comment:
They won't be new at some point in the future. Change to something like `// Optional RSS tag fields`